docs/data/gridftp.md

```diff
@@ -49,10 +49,68 @@ Configuring GridFTP
 To configure which virtual organizations and users are allowed to use your GridFTP server, follow the instructions in
 [the LCMAPS VOMS plugin document](/security/lcmaps-voms-authentication#configuring-the-lcmaps-voms-plugin).
 
+### Set port ranges
+As mentioned above, GridFTP uses port 2811 for control communication as well as a range of ports for the data transfer.
+This range of ports has to defined by setting the variable `GLOBUS_TCP_PORT_RANGE` within the configuration file:
+`/etc/sysconfig/globus-gridftp-server` as it is shown in the next example. This range has to be open within your firewall for inboud communication.
+
+    :::file
+    $GLOBUS_TCP_PORT_RANGE 50000,51000
+
+
+Starting the service
+------------------
+Once you have finished configuring the service and have set the port range, it is time to start the service.
+
+for CentOS 7:
+
+```console
+systemctl start globus-gridftp-server
+```
+
+for CentOS 6:
+```console
+service globus-gridftp-server start
+```
+
+Validating GridFTP
+------------------
```

Comment:
I'd like to move this section back to after the `Managing GridFTP` section and turn it into a step-by-step procedure that validates file transfer as well the gratia probe bits:

> 1. Acquire a user certificate (link to https://opensciencegrid.org/docs/security/user-certs/)
> 1. Find your subject DN (`openssl -in...`)
> 1. Add your DN to `/etc/grid-security/grid-mapfile` and map it to a non-root user (link to https://opensciencegrid.org/docs/security/lcmaps-voms-authentication/#mapping-users)
> 1. As the non-root user, generate your proxy
> 1. Do file transfer (if you have them transfer a file, you should make sure they create it or it's one that definitely already exists)
>
> Also here's how to validate Gratia
>
> 1. ...

Reply:
I think it is easier to understand and work with these two things separately. For me it was easier to configure and test the GridFTP part and then configure and test the Probe. If you still think that it's better to keep it together I will do the change

Reply:
what about leaving the structure like this:
-Configuring GridFTP
-Managing GridFTP (add starting the service here as suggested)
-Validating GridFTP (adding the step by step procedure suggested)
-Monitoring
 -- Enabling GridFTP Gratia Probe
 --Starting the service
 --Validation of GridFTP Gratia Probe

Reply:
hearing none, I just followed your suggestions